west.yml

```diff
@@ -188,7 +188,7 @@ manifest:
       groups:
         - hal
     - name: hal_nordic
-      revision: ee6213b6cfcd7fdc10f7ec3a4f7f37f35dfa5343
+      revision: pull/211/head
```

Comment:
Shouldn't this commit be first?

Reply:
The kconfig option needs to be define first, so first commit will be separation of mode, Kconfig will be used in hal_nordic code.